src/tests/unit/send-get/get_with_offset_1d.f90

```diff
@@ -27,5 +27,5 @@ program get_offset_1d
     enddo
     write(*,*) 'Test passed.'
   endif
-
+sync all
```

Comment:
@afanfa can you please look at this?

@vehre can you please elaborate more on why this is necessary?